src/js/edu-benefits/1990e/helpers.jsx

```diff
@@ -0,0 +1,10 @@
+import { transformForSubmit } from '../../common/schemaform/helpers';
+
+export function transform(formConfig, form) {
```

Comment:
There aren't any transforms, but I'm guessing this function has to be defined for the review-and-submit page to be generated